bin/plugin/commands/changelog.js

```diff
@@ -194,13 +194,15 @@ const REWORD_TERMS = {
  * @return {string[]} Type candidates.
  */
 function getTypesByLabels( labels ) {
-	return uniq(
-		labels
-			.filter( ( label ) =>
-				Object.keys( LABEL_TYPE_MAPPING ).includes( label )
-			)
-			.map( ( label ) => LABEL_TYPE_MAPPING[ label ] )
-	);
+	return [
+		...new Set(
```

Comment:
Out of curiosity, is there any difference between spreading the Set and using [`Array.from`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from)?

Reply:
Good question! I guess the answer has a few different angles:

* Performance - they're almost equal, spreading being a bit faster: https://jsben.ch/5lKjg
* Functionality - they work the same way when working with iterables (which `Set`s are).
* Transpilation - the end result will differ slightly, but that would matter if this code was transpiled (which it isn't).

Unless you have strong feelings, I'd prefer to use spread as it's way more common around the codebase.

Reply:
<img width="827" alt="Screenshot 2022-08-17 at 17 51 24" src="https://user-images.githubusercontent.com/699132/185185269-db32a715-26b1-4fd1-b4eb-c69fe408f57a.png">

Safari, macOS, so I guess it depends but it's nearly the same.

I don't have any objections to using spread here. In general, both approaches discussed are less readable than `uniq`. It isn't immediately obvious that `Set` is used to remove duplicates.

Reply:
> It isn't immediately obvious that `Set` is used to remove duplicates.

I don't know if I agree with that; `[ ...new Set() ]` is a pretty common idiom these days.